psi4/driver/procrouting/dft_funcs/mgga_superfuncs.py

```diff
@@ -161,6 +161,52 @@ def build_mgga_ms2_superfunctional(name, npoints, deriv, restricted):
     return (sup, False)
 
 
+def build_tpss_superfunctional(name, npoints, deriv, restricted):
+
+    # Call this first
+    sup = core.SuperFunctional.blank()
+    sup.set_max_points(npoints)
+    sup.set_deriv(deriv)
+
+    # => User-Customization <= #
+
+    # No spaces, keep it short and according to convention
+    sup.set_name('tpss')
```

Comment:
Names should be upper case to match other (applies to all new funcs). Very surprised this doesn't come from LibXC, looks like they only give TPSSh.

Reply:
Well, that's one of the open questions about libxc standards that hasn't really been addressed. I think opinions vary, but so do functionals. TPSSh could follow other ab initio (non-fitted) functionals like most LDAs and GGAs and some meta-GGAs and have a separate exchange part and correlation part.